.gitignore

```diff
@@ -2,8 +2,9 @@
 titles/rhdh-plugins-reference/build/
 titles/release-notes-rhdh/build/
 titles/install/build
-titles/admin/build
-titles/getting-started-rhdh/
+titles/admin-rhdh/build
+titles/admin
+titles/getting-started-rhdh/build
```

Comment:
[This open PR ](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/106) will take care of all of this. It replaces all of the individual `titles` with a wildcard, which is what we want going forward. It also deletes all of the build files and keeps them from being committed in the future.